low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -0,0 +1,46 @@
+#!/bin/python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com
+
+"""
+ ***************************************************************************
+  Description: Server class to provide fru health and specific information
+ ***************************************************************************
+"""
+
+from resource_map import ResourceMap
+
+
+class ServerMap(ResourceMap):
+    """Provides health information of FRUs in a node"""
+
+    name = "server"
+
+    def __init__(self):
+        """Initialize server"""
+        pass
+
+    def get_health_info(self, rpath):
+        """Get health information of fru in given resource id
+        rpath: Resource id (Example: nodes[0]>compute[0]>hw>disks)
+        """
+        return
+
+
+if __name__ == "__main__":
```

Comment:
this is for testing mainly? Because these will be serving interfaces to DM main and no self fetch & print data is needed. Keep it commented out by default

Reply:
yes, commented now on both server.py and storage.py